check_gluster.rb

```diff
@@ -1,13 +1,4 @@
-#!/usr/bin/ruby -w
-
-# We use old ruby 1.8.7
-# rubocop:disable Style/HashSyntax
-
-# We match all exceptions except SystemExit to redefine exit status for nagios
-# rubocop:disable Lint/RescueException
```

Comment:
Removing rubocop lines causing a lot of warnings while checking code with rubocop. Can we leave these rubocop custom configuration?